README.md

```diff
@@ -11,6 +11,7 @@ Users of the bot may encounter error codes when attempting to execute slash-comm
 * `E1022` - The CSS Discord server does not contain a role with the name "Guest" (required for the `/induct`, `/makemember`, `/stats` & `/archive` commands and the `kick_no_introduction_members` & `introduction_reminder` tasks)
 * `E1023` - The CSS Discord server does not contain a role with the name "Member" (required for the `/makemember` command)
 * `E1024` - The CSS Discord server does not contain a role with the name "Archivist" (required for the `/archive` command)
+* `E1025` - The CSS Discord server does not contain a role with the name "Applicant" (required for the `/induct` command)
```

Comment:
Maybe re-word this error? This comes across like you ***must*** have the applicant role or the command won't work which I think we agreed not to do?